ci/ci

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import platform
+import argparse
+import subprocess
+import fileinput
+import shlex
+import time
+
+pbs_dirname = ''
+
+
+def log_error(str_to_print):
+    t = time.localtime()
+    current_time = time.strftime("%H:%M:%S", t)
+    print('\033[91m' + current_time + " ---> " + str_to_print + '\033[0m')
+
+
+def log_info(str_to_print):
+    t = time.localtime()
+    current_time = time.strftime("%H:%M:%S", t)
+    print('\033[1m' + current_time + " ---> " + str_to_print + '\033[0m')
+
+def run_cmd(cmd, return_output=False):
+    cmd = shlex.split(cmd)
+    try:
+        a = subprocess.Popen(cmd,stdout=subprocess.PIPE)
+        out,err = a.communicate()
+        if a.returncode != 0:
+            log_error("command failed")
+            log_error(str(err))  
+        else:
+            if return_output:
+                return str(out)
+    except Exception as e:
+        log_error("The command failed.")
+        log_error(e)
+
+
+def run_docker_cmd(cmd):
+    try:
+        docker_cmd = shlex.split(cmd)
+        a = subprocess.Popen(docker_cmd)
+        a.communicate()
+        if a.returncode != 0:
+            log_error(
+                "docker cmd returned with non zero exit code, dropping you into container")
+            docker_cmd = shlex.split(
+                "docker-compose exec pbs-build bash -c \'cd /pbspro && /bin/bash\'")
+            subprocess.run(docker_cmd)
+    except Exception as e:
+        log_error("Failed\n:")
+        log_error(e)
+
+
+def write_to_file(file_path, value):
+    with open(file_path, "w+") as f:
+        f.write(value)
+
+
```

Comment:
why 2 lines instead of 1?

Reply:
Running pep8 on the file produces 2 lines between functions.